esphome/components/atm90e32/sensor.py

```diff
@@ -0,0 +1,98 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import sensor, spi
+from esphome.const import \
+    CONF_ID, CONF_POWER, CONF_FREQUENCY, \
+    ICON_FLASH, UNIT_HZ, UNIT_VOLT, UNIT_AMPERE, UNIT_WATT
+
+CONF_VOLTAGE_A = 'voltage_a'
+CONF_VOLTAGE_B = 'voltage_b'
+CONF_VOLTAGE_C = 'voltage_c'
+CONF_CURRENT_A = 'current_a'
+CONF_CURRENT_B = 'current_b'
+CONF_CURRENT_C = 'current_c'
+CONF_LINE_FREQ = 'line_freq'
```

Comment:
```suggestion
CONF_LINE_FREQUENCY = 'line_freqquency'
```

ESPHome prefers long_but_explicit names